src/challenges/defi-challenges/simple-lending.mdx

```diff
@@ -0,0 +1,244 @@
+---
+name: Simple Lending
+index: 3
```

Comment:
Can we update this to index "4"? So, this follows after recent challenge from 0x.